LayoutTests/accessibility/mac/line-text-marker-range-in-list-item.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../../resources/accessibility-helper.js"></script>
+<script src="../../resources/js-test.js"></script>
+</head>
+<body>
+
+<ul id="list">
+  <li id="listitem">Once upon a time</li>
+</ul>
+
+<script>
+  let output = "This tests that fetching the string for the line range corresponding to a text marker fetches the whole string, and it doesn't get truncated due to the list marker adding extra characters\n\n";
```

Comment:
I think we go for 4-space indentation of JS in accessibility tests (and we do that below too, so it would be good to be consistent here)